src/NUnitFramework/framework/SpecialValue.cs

```diff
@@ -33,7 +33,7 @@ public enum SpecialValue
     {
         /// <summary>
         /// Null represents a null value, which cannot be used as an 
-        /// argument to an attriute under .NET 1.x
+        /// argument to an attribute under .NET 1.x
```

Comment:
My guess is that we can probably get rid of SpecialValue entirely, but since this PR was about typos I didn't want to confuse the issue.